apps/site/pages/en/index.mdx

```diff
@@ -14,29 +14,19 @@ layout: home
 
   </div>
 
-  <div>
-    <WithNodeRelease status={['LTS']}>
-      {({ release }) => (
-        <>
-          <DownloadButton release={release}>Download Node.js (LTS)</DownloadButton>
-          <small>
-            Downloads Node.js <b>{release.versionWithPrefix}</b>
-            <sup title="Downloads a Node.js installer for your current platform">1</sup> with long-term support.
-            Node.js can also be installed via <Link href="/download">version managers</Link>.
-          </small>
-        </>
-      )}
-    </WithNodeRelease>
-
-    <WithNodeRelease status={['Current']}>
-      {({ release }) => (
-        <small>
-          Want new features sooner?
-          Get <b>Node.js <DownloadLink release={release}>{release.versionWithPrefix}</DownloadLink></b>
-          <sup title="Downloads a Node.js installer for your current platform">1</sup> instead.
-        </small>
-      )}
-    </WithNodeRelease>
+  <div className="flex gap-4">
+
+    <div className="flex flex-col gap-2">
+      <Button kind="special" className="!hidden dark:!block" href="/download">Install Node.js</Button>
+
+      <Button kind="primary" className="!block dark:!hidden" href="/download">Install Node.js</Button>
+
+      <Button kind="secondary" className="!block" href="https://www.herodevs.com/support/node-nes">
```

Comment:
```suggestion
      <Button kind="secondary" className="!block" href="https://www.herodevs.com/support/node-nes?utm_source=NodeJS+&utm_medium=Link&utm_campaign=Homepage_button">
```
Unless we hear otherwise from @joeeames, let's roll with these UTMs?

Reply:
I won't roll with random UTM params.

Reply:
Why is this link on the homepage?

It doesn’t appear to be affiliated with the node foundation.

Reply:
What link?

Reply:
If you're referring to the HeroDevs link; https://www.herodevs.com/blog-posts/herodevs-joins-openjs-foundations-ecosystem-sustainability-program-as-first-partner